charmander/models/order_event_model.py

```diff
@@ -25,7 +25,7 @@ class OrderEvent(UnixTimestampEntityMixin, metaclass=EventMeta):
     __container__ = Order
     __unique_parent__ = True
 
-    customer_id = Column(UUIDType, nullable=False)
+    customer_id = Column(UUIDType, nullable=True)
```

Comment:
We could inherit this from the parent, but it seemed simpler just to make it optional. The `OrderInitialized` event still requires this property so we know that the event history has it and we can put it in a roll up if we want to.